cli/src/commands/decompress-sol/index.ts

```diff
@@ -0,0 +1,64 @@
+import { Command, Flags } from "@oclif/core";
+import {
+  CustomLoader,
+  defaultSolanaWalletKeypair,
+  generateSolanaTransactionURL,
+  getSolanaRpcUrl,
+} from "../../utils/utils";
+import { Connection, PublicKey, LAMPORTS_PER_SOL } from "@solana/web3.js";
+import { Rpc, decompressLamports } from "@lightprotocol/stateless.js";
+
+class MintToCommand extends Command {
+  static summary = "Decompress SOL.";
+
+  static examples = ["$ light decompress-sol --to PublicKey --amount 10"];
+
+  static flags = {
+    to: Flags.string({
+      description: "Specify the recipient address.",
+      required: true,
+    }),
+    amount: Flags.integer({
+      description: "Amount to mint, in SOL.",
+      required: true,
+    }),
+  };
+
+  static args = {};
+
+  async run() {
+    const { flags } = await this.parse(MintToCommand);
+    const to = flags["to"];
+    const amount = flags["amount"];
+    if (!to || !amount) {
+      throw new Error("Invalid arguments");
+    }
+
+    const loader = new CustomLoader(`Performing compress-sol...\n`);
+    loader.start();
+
+    try {
+      const toPublicKey = new PublicKey(to);
+      const payer = defaultSolanaWalletKeypair();
+
+      const connection = new Connection(getSolanaRpcUrl());
```

Comment:
needs to be `getTestRpc` / new `TestRpc`() instance, which extends `Connection` with the compression API methods.
(we'll then independently have to replace `TestRpc` use with `Rpc`  once we've successfully tested Rpc class with local photon+sdk)